monsoft-crm-app/bases/ai/utils/ai-model-to-sdk-model.util.ts

```
@@ -0,0 +1,17 @@
+import { LanguageModelV1 } from 'ai';
+
+import { Success, Error } from '@errors/utils';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)